data_diff/format.py

```diff
@@ -40,6 +40,7 @@ def jsonify(
     dataset2_columns: Columns,
     columns_diff: Dict[str, List[str]],
     with_summary: bool = False,
+    stats_only: bool = False,
```

Comment:
Do you mind adding a simple test case [here](https://github.com/datafold/data-diff/blob/main/tests/test_format.py#L11)? At least one checking that `rows` is `None` when `stats_only=True`.

Reply:
> I wasn't able to get the tests to run locally on a M1 mac, sadly. Would be fantastic if there was a dockerised setup one could run end-to-end for that

Looks like they're having trouble running the tests. I can spend some time tomorrow adding one

Reply:
added a test case